pkgs/by-name/ca/caido/package.nix

```
@@ -0,0 +1,39 @@
+{ lib, fetchurl, appimageTools, makeWrapper }:
+
+let
+  pname = "caido";
+  version = "0.29.2";
+  name = "caido-desktop-linux-v${version}-e0f8102b";
+  nameExecutable = pname;
+  src = fetchurl {
+    url = "https://storage.googleapis.com/caido-releases/v${version}/${name}.AppImage";
+    name = "caido-${version}.AppImage";
+    sha256 = "sha256-4PgQK52LAX1zacmoUK0muIhrvFDF7anQ6sx35I+ErVs=";
```

```suggestion
    hash = "sha256-4PgQK52LAX1zacmoUK0muIhrvFDF7anQ6sx35I+ErVs=";
```
Use hash instead of sha256.